### PR TITLE
BUG: fix f2py pep338 execution method

### PR DESCRIPTION
**Pull request by juliantaylor**
Closes gh-12576

### CODE REVIEW

**Comment by mattip** [comment]
it is strange that not a single test invokes f2py this way

File: numpy/f2py/__main__.py [hunk 1]
```diff
@@ -1,6 +1,4 @@
 # See http://cens.ioc.ee/projects/f2py2e/
-from __future__ import division, print_function
```
[the remark this answers is not in the document]
Comment (juliantaylor) [reply numpy/f2py/__main__.py]: ops didn't want to remove it, I though I pushed the version with them in, added ack

**Comment by charris** [comment]
> it is strange that not a single test invokes f2py this way

Want to put a test together?  `numpy/tests/test_scripts.py` is probably the best place.

**Comment by charris** [comment]
LGTM Python looks stuck, as is often the case, so putting this in. Thanks Julian.

**Pull request merged**